.ci/build.sh

```diff
@@ -5,7 +5,15 @@ SCRIPT_DIR="$(dirname "$0")"
 set -exuo pipefail
 source "$SCRIPT_DIR/common"
 
-docker build -t $CI_IMAGE_URI --build-arg IMAGE_REPOSITORY=${IMAGE} --build-arg IMAGE_TAG=${COMMIT} "$SCRIPT_DIR/.."
+OUTFILE="$(mktemp -d)/image.tar"
+
+docker buildx build -o type=oci,dest=${OUTFILE} \
+  -t $CI_IMAGE_URI \
+  --platform=${CI_IMAGE_PLATFORMS} \
```

Comment:
Should we build only x86 for ci tests here? We can build arm64 in another job otherwise arm64 is delaying the entire test pipeline. And in the future we could do testing for arm64 in parallel as well.

Reply:
Yeah will do it after fixing image load problem

Reply:
Fixed